x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A common set of SGML entities and XML/CSS style sheets"
+DESCRIPTION="This package provides a common set of SGML entities and XML/CSS \
+style sheets used in building/formatting the documentation provided in other\
+X.Org packages.   It's typically only needed by people building from \
+source who want to produce formatted documentation from their builds, \
+or those who have installed the HTML version of the documentation, \
+which refers to the included common xorg.css stylesheet."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/proto/"
```

Comment:
```diff
-HOMEPAGE="http://xorg.freedesktop.org/releases/individual/proto/"
+HOMEPAGE="https://xorg.freedesktop.org/releases/individual/proto/"
```

Reply:
This should be the real homepage: https://www.x.org/releases/individual/doc/

Reply:
We could also use both :)
```
HOMEPAGE="https://www.x.org/releases/individual/doc/
	https://xorg.freedesktop.org/releases/individual/proto/"
```
But I guess I'm ok with any single line `HOMEPAGE` too.

Reply:
hmm ... I'm guessing */doc should be used (as */proto links to the proto related downloads)
Or just "https://www.xorg.com"?